index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Chris Duke!!!</title>
+
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.3/css/font-awesome.css">
+  <link rel="stylesheet" href='https://fonts.googleapis.com/css?family=Raleway'
+</head>
+<body>
+  <nav class="top-nav">
+    <ul class="nav-list">
+      <li class="nav-list__item nav-list__item--brand">My Portfolio</li>
+      <li class="nav-list__item">About</li>
+      <li class="nav-list__item">Day One</li>
+      <li class="nav-list__item">Cool Stuff</li>
+    </ul>
+  </nav>
+  <header class="hero">
+    <div class="section-container hero__container">
+      <h2 class="hero__title">This is my portfolio!</h2>
+      <hr class="divider divider--hero">
+      <p class="hero__text">Please enjoy.</p>
+    </div>
+  </header>
+  <section class="bg-pop">
+    <div class="nav-list__item">
```

Comment:
Directly within the sections will be the `section-container` which sets the max width and centers the content